ui/imports/shared/views/ProfileView.qml

```diff
@@ -362,7 +362,7 @@ Rectangle {
             input.multiline: true
             minimumHeight: 152
             maximumHeight: 152
-            placeholderText: qsTr("Ask a question that only the real %1 will be able to answer e.g. a question about a shared experience, or ask Mark to enter a code or phrase you have sent to them via a different communication channel (phone, post, etc...).").arg(userIsEnsVerified ? userEnsName : userDisplayName)
+            placeholderText: qsTr("Ask a question that only the real %1 will be able to answer e.g. a question about a shared experience, or ask %1 to enter a code or phrase you have sent to them via a different communication channel (phone, post, etc...).").arg(userIsEnsVerified ? userEnsName : userDisplayName)
```

Comment:
shouldn't `userDisplayName` do the job?

Reply:
Yes, it should be used here

Reply:
> shouldn't `userDisplayName` do the job?

Do you mean that `userDisplayName` already contains the ENS name if it is the case? Or no need to visualize ENS name in any case?

Reply:
> > shouldn't `userDisplayName` do the job?
> 
> Do you mean that `userDisplayName` already contains the ENS name if it is the case? Or no need to visualize ENS name in any case?

Yup, just use `userDisplayName` (you can test in our official channel, some ppl have an ENS name, like `@magnus` or `@sitorin`)

Reply:
Ok!! Will update it then!! Thanks for the clarification!